internal/cmd/policy/updateperson/updateperson.go

```diff
@@ -0,0 +1,161 @@
+// Copyright The gittuf Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package updateperson
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/gittuf/gittuf/experimental/gittuf"
+	trustpolicyopts "github.com/gittuf/gittuf/experimental/gittuf/options/trustpolicy"
+	"github.com/gittuf/gittuf/internal/cmd/common"
+	"github.com/gittuf/gittuf/internal/cmd/policy/persistent"
+	"github.com/gittuf/gittuf/internal/policy"
+	"github.com/gittuf/gittuf/internal/tuf"
+	tufv02 "github.com/gittuf/gittuf/internal/tuf/v02"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	p                    *persistent.Options
+	policyName           string
+	personID             string
+	publicKeys           []string
+	associatedIdentities []string
+	customMetadata       []string
+}
+
+func (o *options) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&o.policyName,
+		"policy-name",
+		policy.TargetsRoleName,
+		"name of policy file to update person in",
+	)
+
+	cmd.Flags().StringVar(
+		&o.personID,
+		"person-ID",
+		"",
+		"person ID to update",
+	)
+	cmd.MarkFlagRequired("person-ID") //nolint:errcheck
+
+	cmd.Flags().StringArrayVar(
+		&o.publicKeys,
+		"public-key",
+		[]string{},
+		"public keys for person (replaces existing keys)",
```

Comment:
I think we can omit this given the `Long` description informing the user of this.
```suggestion
		"public keys for person",
```